parquet-hadoop/src/main/java/org/apache/parquet/ParquetReadOptions.java

```diff
@@ -44,6 +44,8 @@ public class ParquetReadOptions {
   private static final int ALLOCATION_SIZE_DEFAULT = 8388608; // 8MB
   private static final boolean PAGE_VERIFY_CHECKSUM_ENABLED_DEFAULT = false;
   private static final boolean BLOOM_FILTER_ENABLED_DEFAULT = true;
+  // Default to true if JDK 17 or newer.
```

Comment:
Don't quite understand this comment. Where it is set to true?

Reply:
Oops. Comment got left behind from the original. I changed the initialization after some review comments.